### ISSUE TITLE
Simulate policy returns FrozenOrderedDict instead of dict

### ISSUE THREAD

**Issue by sandrich** [awslabs/aws-service-catalog-puppet#519]
I get the following error when running a custom policy simulator

```
Invalid type for parameter ContextEntries[0], value: {'ContextKeyName': 'ec2:InstanceType', 'ContextKeyType': 'string', 'ContextKeyValues': ('t2.micro',)}, type: <class 'luigi.freezing.FrozenOrderedDict'>, valid types: <class 'dict'>
```

As an example with the following policy simulator definitio

```
deny-non-micro-instances:
    simulation_type: "custom"
    policy_input_list: {"Version":"2012-10-17","Statement":[{"Sid":"RequireMicroInstanceType","Effect":"Deny","Action":"ec2:RunInstances","Resource":"*","Condition":{"StringNotEquals":{"ec2:InstanceType":["t2.micro","t3.micro","t3a.micro"]}}},{"Sid":"AllowEC2Launches","Effect":"Allow","Action":"*","Resource":"*"}]}
    action_names:
      - "ec2:RunInstances"
    context_entries: 
      - ContextKeyName: ec2:InstanceType
        ContextKeyValues:
          - t2.medium
        ContextKeyType: string
    expected_decision: "implicidDeny"
    simulate_for:
      tags:
        - tag: "type:prod"
          regions: "default_region"
```

**Comment by eamonnfaherty** [comment]
Thanks for reporting this.  Would it be possible to get the full stack trace please.

**Comment by eamonnfaherty** [comment]
this is resolved in 0.171.0
